uint8 motor_id
uint8 command_id
uint16 value
